pipelinesScripts/validator/dependency/dependency.go

```diff
@@ -45,6 +45,15 @@ func GetJfrogLatest() (dependencies []Details, err error) {
 	return
 }
 
+func IsCoreVersionOneReleased(dependencies []Details) bool{
```

Comment:
IsCoreVersionOneReleased --> IsCoreV1DepIncluded

Let's also add a clear description for this function.